src/main/java/practice/StreamPractice.java

```diff
@@ -14,7 +19,16 @@
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        int result = numbers.stream()
+                .map(s -> s.split(","))
+                .flatMap(a -> Arrays.stream(a).sequential())
```

Comment:
```suggestion
                 .flatMap(n -> Arrays.stream(n.split(",")))
```
тут можна до однієї операції спростити